index.bs

```diff
@@ -306,22 +318,24 @@ disabled for a given browsing context.
 <div algorithm>
 
 To determine if an <dfn export>event is enabled</dfn> given |session|,
-|event name| and |browsing context|:
+|event name| and |browsing contexts|:
```

Comment:
Making *browsing contexts* plural here is probably in the service of an anticipated future need, but I wonder what it is? Right now, it will either be an empty set or a set of one thing, the possible  parent browsing context for the context being created or destroyed.

The behavior here is to treat the event as enabled if it's enabled in any of the given browsing contexts, but when will be be passing in multiple browsing contexts here?

This also had the side effect making a browsing context optional, but if that's the main purpose maybe we can make it explicitly optional, but still singular?

Reply:
It's for e.g. shared workers. I actually backported this change from the realms work just to avoid pointless conflicts.

Reply:
Ah, shared workers. Is that the only known case, so that for an implementation that doesn't support shared workers, this would always be either an empty set or a set of one? If so, adding a comment to that effect would be nice for those implementers.